### DISCUSSION TITLE
Installing and Running Artisan in Raspberry Pi 4B with Raspberry Pi OS 64 Bits

### DISCUSSION THREAD

**Discussion by dhionnystrauss** [artisan-roaster-scope/artisan#1768] [Q&A]
Hello, I am also new here. It is also highly interesting for Coffee Roasters! Congrats.
I downloaded the Raspberry Pi version 64 Bits and tried to install it using my Raspberry PI 4B with 4 GB RAM and Pi OS 64 Bits, but it is giving me an installation error. Do you have a tutorial available on how to install and Run Artisan on Raspberry Pi 4B or even for Raspberry Pi 5?
Thank you so much,
Tommy

**Comment by mrpenner** [comment]
Is it the same as https://github.com/artisan-roaster-scope/artisan/issues/1765 ?

**Comment by dhionnystrauss** [comment]
I tried to install it in My Raspberry Pi 4B PiOS 64 Bits, following the steps on https://artisan-scope.org/docs/installation/ and I finally could install Okay. Now, I will proceed to test it. Thank you so much and keeping in contact. Thank you so much and Merry Christmas 2024!
![20241225_224320](https://github.com/user-attachments/assets/6b3c942d-e3bd-4ff3-8237-7fecd08af11d)
